src/cracker.c

```diff
@@ -279,7 +279,8 @@ void crack()
 			case KEY_REJECTED:
 				fail_count = 0;
 				pin_count++;
-				advance_pin_count();
+				if (!get_pin_string_mode())
```

Comment:
wouldnt this lead to "quitting after 0 pin attempts" when using string pin ?

Reply:
`advance_pin_count()` is for to increment the index of p1 or p2 on normal attack.
I know that in string mode has protection of `set_max_pin_attempts(1)` implicit and also does not save .wpc file to avoid wrong.
To avoid any change in the index of p1 and/or p2 when using string pin, the use of if condition is good for it.